src/migrate.ts

```diff
@@ -42,55 +42,94 @@ export async function migrate (args: ArgumentsCamelCase<Params>): Promise<void>
   }
 
   const numKeys = await sbp('chelonia.db/keyCount')
+  let numMigratedKeys = 0
   let numVisitedKeys = 0
-  let shouldExit = 0
-
-  const handleSignal = (signal: string, code: number) => {
-    process.on(signal, () => {
-      console.error(`Received signal ${signal} (${code}). Finishing current operation.`)
-      // Exit codes follow the 128 + signal code convention.
-      // See <https://tldp.org/LDP/abs/html/exitcodes.html>
-      shouldExit = 128 + code
-    })
+
+  const reportStatus = () => {
+    console.log(`[chel] ${colors.green('Migrated:')} ${numMigratedKeys} entries`)
```

Comment:
Why are various log statements here prefixed with `[chel]`?

Don't we know that this is the chel command being run?

Reply:
Not sure, that's how it was before